packages/storage-ui/src/Components/Pages/CidsPage.tsx

```diff
@@ -233,6 +262,20 @@ const CidsPage = () => {
           </TableBody>
         </Table>
       </div>
+      {!!pins.length &&
```

Comment:
I feel we should hide it when there's just 1 page.
```suggestion
      {!!pins.length && pins.length > 1 &&
```